src/components/image/img-with-placeholder.js

```diff
@@ -0,0 +1,140 @@
+import { getSrcsetString } from '../../utils/image'
+import get from 'lodash/get'
+import Placeholder from '../../../assets/svg/img-loading-placeholder.svg'
+import predefinedPropTypes from '../../constants/prop-types'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+const _ = {
+  get,
+}
+
+const ImgContainer = styled.div`
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+  padding-bottom: ${props => `${props.heightWidthRatio * 100}%`};
+`
+
+const StyledPlaceholder = styled(Placeholder)`
```

Comment:
Since we are passing `toShow` to `StyledPlaceholder`, which causes this warning on the browser.
`Warning: React does not recognize the `toShow` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `toshow` instead. If you accidentally passed it from a parent component, remove it from the DOM element.`

Maybe we should use composition instead.
```
<StyledPlaceholder>
  <Placeholder/>
</StyledPlaceholder>
```

Reply:
👍